### PR TITLE
Prepare a package for publishing

### PR DESCRIPTION
**Pull request by anton-ptashnik**
This PR intends to assist with publishing the package to PyPI by:
- making some changes to package metadata
- adding a GitHub action for publishing a package easily from GitHub
- providing steps needed to publish a package
- address #5

# Initial setup to allow package publishing

1. Login to https://pypi.org/ (create an account if missing)
2. Generate a token
2.1. Go to `Account Settings` -> `API tokens`
2.2. Create a new token. Name - any, Scope - entire account 
3. Put the token into GitHub secrets for the repo
3.1. Go to the repo settings -> `Secret and Variables`
3.2. Create a new secret with name=PYPI_TOKEN and value=generated token

# Steps to publish a package

Note these steps need to be done after the PR is merged so release is done on this original repo rather than on the fork.
1. Go to `Releases` of the repo
2. Create a new release:
2.1. Create a new version tag, v0.1.0 for the first release
2.2. Press `Generate release notes`. It makes release name == tag name and puts a default description
2.3. Optional. Modify the description to explain changes better if needed  
2.4. Submit

Once done, GitHub starts a publish action and the package will appear on PyPI in a few seconds

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 Use the GoDice Python API to integrate GoDice functionality into your own Python applications
 
+![PyPI - Version](https://img.shields.io/pypi/v/godice)
```
Comment (anton-ptashnik) [inline README.md:8]: a badge will render correctly after publishing the package

File: pyproject.toml [hunk 2]
```diff
@@ -4,7 +4,17 @@ build-backend = "flit_core.buildapi"
 
 [project]
 name = "godice"
-authors = [{name = "Particula", email = "iavtomator@gmail.com"}]
+authors = [
```
Comment (anton-ptashnik) [inline pyproject.toml:7]: @Geek5510 please update/extend authors list as needed. I recognize my work was based on knowledge of others about device API

File: pyproject.toml [hunk 3]
```diff
@@ -4,7 +4,17 @@ build-backend = "flit_core.buildapi"
...
+    {name = "Guy Levi", email = "todo_update@it.com"},
+    {name = "Anton Ptashnik", email = "iavtomator@gmail.com"},
+]
+license = {file = "LICENSE.md"}
```
Comment (anton-ptashnik) [inline pyproject.toml:11]: @Geek5510 I'm not strong in licensing but I'm almost sure a published package should have a correct one. I see the repo has some custom license while open source projects usually have MIT or similar one.
Please review if the current license still fits and make changes if needed

**Comment by Geek5510** [comment]
Thank you for your suggestions and work! we'll check about the licensing and see what can we do

**Pull request merged**